tests/integration-tests/utils.py

```diff
@@ -802,3 +839,27 @@ def is_fsx_supported(region: str):
 
 def is_directory_supported(region: str, directory_type: str):
     return False if "us-iso" in region and directory_type == "SimpleAD" else True
+
+
+def set_protected_failure_count(remote_command_executor, protected_failure_count):
+    """Disable protected mode by setting protected_failure_count to -1."""
```

Comment:
Minor: the comment is not aligned to what the function is doing.

This would be the comment of the disable_protected_mode below.
Minor: we can have these two functions close to each other.

Reply:
fixed